examples/elvis-operator/elvis_operator.bal

```diff
@@ -4,13 +4,11 @@ public function main() {
     string|() x = ();
 
     // If you need to get the `string` value of x, and if the value of x is `nil`, you may want
-    // to assign a known value. This is how you can do it via type-guard (also known as `is` check)
-    // expression.
+    // to assign a known value. This is how you can do it via the type-guard.
```

Comment:
```suggestion
    // to assign a value. This is how you can do it via the type-guard.
```
Isn't it enough to say a "value"? Or, do we specifically need to say a "known value"?
If we are referring to a single element it is correct to use the hyphen here (i.e., "type-guard"). Else, if we are referring to a type called "guard", it is incorrect to use the hyphen here. Then, it is better to say: the type `guard`.

Reply:
We seem to have used type-guard in general.